### PR TITLE
Minor changes for fitlins

### PR DESCRIPTION
**Pull request by adelavega**
Two minor changes for latest `fitlins` update:

- Update Dockerfile to work with jupyterhub shim
- Don't set bids config

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
 # Copy the current directory contents into the container
 COPY [".", "/src/neuroscout"]
 
+USER root
+RUN chown -R neuro /src
+USER neuro
+
 RUN /bin/bash -c "source activate neuro \
-      && pip install -q --no-cache-dir -e /src/neuroscout/" \
+      && pip install --no-cache-dir -e /src/neuroscout/" \
```
Comment (effigies) [inline Dockerfile]: Removing `-q` was only for debug. You could put it back in, if you like.

File: Dockerfile [hunk 2]
```diff
@@ -1,29 +1,28 @@
-# Use an poldracklab/fitlins as a parent image
 FROM poldracklab/fitlins
```
Comment (effigies) [inline Dockerfile]: Do you want this to build off of `:master` or `:latest`?

File: Dockerfile [hunk 3]
```diff
@@ -3,26 +3,19 @@ FROM poldracklab/fitlins
...
-
 RUN git config --global user.name "Neuroscout"
 RUN git config --global  user.email "user@example.edu"
 
 # Copy the current directory contents into the container
 COPY [".", "/src/neuroscout"]
 
-USER root
-RUN chown -R neuro /src
-USER neuro
```
Comment (effigies) [inline Dockerfile]: I think you need this in order to `pip install -e` below.

Comment (adelavega) [reply Dockerfile]: I actually just kept the user at `root`. It didn't seem to cause any problems for running the `jupyterhub` shim. Any other problems I should be aware of?

Comment (effigies) [reply Dockerfile]: Oh, okay. Not that I know of.

**Pull request merged**